apps/gov-website/pages/api/identity/judgement.tsx

```diff
@@ -75,3 +89,35 @@ export default withMethodGuard(
 	},
 	["POST"]
 );
+
+const assignDiscordRole = (discordUsername: string) => {
+	const [username, discriminator] = discordUsername.split("#");
```

Comment:
should be an async function that grabs a bot from `getDiscordBot(DISCORD_BOT.Token)`